etc/contracts-test-data/scripts/build.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import {
+    needsRecompilation,
+    setCompilationTime,
+    isFolderEmpty,
+    CONTRACTS_DIR,
+    OUTPUT_DIR,
+    TIMESTAMP_FILE
+} from './utils';
+import { exec } from 'child_process';
+
+async function main() {
+    const timestampFilePath = path.join(process.cwd(), TIMESTAMP_FILE); // File stores the timestamp of last compilation
```

Comment:
you are repeating this twice (as it already exists in the other build.rs) - maybe move it to a shared util instead?

Reply:
That's right, I was thinking of it, but as these are different workspaces, I was not sure if a shared util is a good idea